assignment7.txt

```diff
@@ -0,0 +1,24 @@
+Bettina Benitez
+
+1. ++*p reads right to left. Therefore, it increments point p.  Same with *++p, the order of operation is right to left. But point p is incremented then the dereferenced. *p++ is read left to right therefore it is incremented then dereferenced. 
+
+2. No the order of precedence is not guaranteed
+
+3. advantages:
+- it points to a specific lovation in memory
+- provides direct access to memory
+- reduces storage and complexity
+
+4
+ - 4.1: *char
+ - 4.2: invalid
```

Comment:
This is actually valid! "xyz" is just an array of characters, so the "xyz"[1] is just accessing "y". Then you just subtract 'y' from it to get 0. This is because, if you can recall, a char is just a value mapped to a character!